2-storage.yml

```diff
@@ -5,4 +5,3 @@ events:
     parameters:
       - name: message
         type: ByteString
-
```

Comment:
?

Reply:
This yaml had two empty lines at the end, I removed one of them.